test/sql/copy/parquet/kv_metadata.test

```diff
@@ -0,0 +1,18 @@
+# name: test/sql/copy/parquet/kv_metadata.test
+# group: [parquet]
+
+require parquet
+
+statement ok
+COPY (SELECT 1, 'foo') TO  '__TEST_DIR__/kv_metadata_test.parquet' (FORMAT PARQUET, KV_METADATA {foo: 'bar', baz: 42});
+
+query II
+SELECT key::VARCHAR, value::VARCHAR FROM parquet_kv_metadata('__TEST_DIR__/kv_metadata_test.parquet');
```

Comment:
Perhaps also (1) try the function on a file without metadata, and (2) try the function with a glob/list of multiple Parquet files ('data/parquet/*.parquet')

Reply:
Im just hooking into the existing parquet metadata reading code, so I assumed this all would work if it works for the normal metadata, but sure I can add some tests for KV specifically.